pkg/acsengine/defaults-cloud-controller-manager.go

```diff
@@ -0,0 +1,69 @@
+package acsengine
+
+import (
+	"strconv"
+
+	"github.com/Azure/acs-engine/pkg/api"
+)
+
+func setCloudControllerManagerConfig(cs *api.ContainerService) {
+	o := cs.Properties.OrchestratorProfile
+	staticLinuxCloudControllerManagerConfig := map[string]string{
+		"--allocate-node-cidrs": strconv.FormatBool(!o.IsAzureCNI()),
+		"--cloud-provider":      "azure",
+		"--cloud-config":        "/etc/kubernetes/azure.json",
+		"--cluster-cidr":        o.KubernetesConfig.ClusterSubnet,
+		"--kubeconfig":          "/var/lib/kubelet/kubeconfig",
+		"--leader-elect":        "true",
+		"--v":                   "2",
+	}
+
+	// Set --cluster-name based on appropriate DNS prefix
+	if cs.Properties.MasterProfile != nil {
+		staticLinuxCloudControllerManagerConfig["--cluster-name"] = cs.Properties.MasterProfile.DNSPrefix
+	} else if cs.Properties.HostedMasterProfile != nil {
+		staticLinuxCloudControllerManagerConfig["--cluster-name"] = cs.Properties.HostedMasterProfile.DNSPrefix
+	}
+
+	staticWindowsCloudControllerManagerConfig := make(map[string]string)
+	for key, val := range staticLinuxCloudControllerManagerConfig {
+		staticWindowsCloudControllerManagerConfig[key] = val
+	}
+	// Windows cloud-controller-manager config overrides
+	// TODO placeholder for specific config overrides for Windows clusters
+
+	// Default cloud-controller-manager config
+	defaultCloudControllerManagerConfig := map[string]string{
+		"--node-monitor-grace-period": DefaultKubernetesCtrlMgrNodeMonitorGracePeriod,
```

Comment:
From the md file, the default is "--route-reconciliation-period", a typo?

Reply:
Good catch! Actually, the doc is correct, it's an error in the `defaultCloudControllerManagerConfig` object! (see the changes to `parts/k8s/manifests/kubernetesmaster-cloud-controller-manager.yaml`). I'll fix.